amundsen_application/static/js/pages/TableDetailPage/index.tsx

```diff
@@ -396,7 +403,12 @@ export class TableDetail extends React.Component<
                   )}
                 </section>
                 <section className="right-panel">
-                  <EditableSection title={Constants.OWNERS_TITLE}>
+                  <EditableSection
+                    title={Constants.OWNERS_TITLE}
+                    readOnly={!data.is_editable}
```

Comment:
What does `data.is_editable` currently represent and how is it currently set? I assume it represents whether or not the entire table is editable, have we switched to as concept of setting `is_editable` on the table data and using that to effect *both* descriptions and owners at the same time?